File: 1-codes/micro/micro10.c
#import<stdio.h>

int fatorial(int n) {
  if (n <= 0) {
    return 1;
  } else {
    return (n * fatorial(n - 1));
  }
}

int main() {
  int numero, fat;

  printf("Digite um numero: ");
  scanf("%d", &numero);

  fat = fatorial(numero);
  printf("O fatorial de %d e %d", numero, fat);

  return 0;
}

File: 1-codes/micro/micro10.py
def micro10():
    numero = 0
    fat = 0
    print("Digite um numero: ")
    numero = int(input())
    fat = fatorial(numero)
    print("O fatorial de " + str(numero) + " e " + str(fat))

def fatorial(n):
    if n <= 0:
        return 1
    else:
        return (n * fatorial(n-1))

micro10()
